pytest_reportportal/service.py

```diff
@@ -60,9 +60,9 @@ def async_error_handler(self, exc_info):
     def _stop_if_neccessary(self):
         try:
             exc, msg, tb = self._errors.get(False)
-            traceback.print_exception(exc, msg, tb)
+            err = traceback.format_exception(exc, msg, tb)
```

Comment:
if ignore_errors is True, then no traceback is displayed? Errors are silently ignored.
Is this desired behaviour?